api/shared.thrift

```diff
@@ -10,12 +10,14 @@ enum ErrorCode {
   GENERAL,
   AUTH_DENIED,  // Authentication denied. We do not allow access to the service.
   UNAUTHORIZED, // Authorization denied. User does not have right to perform an action.
-  API_MISMATCH  // The client attempted to query an API version that is not supported by the server.
+  API_MISMATCH, // The client attempted to query an API version that is not supported by the server.
+  SOURCE_FILE,  // The client send a source code which contains errors.
```

Comment:
What is the purpose of this enum constant? CodeChecker is a tool that collects source files that contain "errors", more or less... so usually we should expect the client to send source files in which bug reports are present.